app/definition/ApiDefinitionFactory.scala

```diff
@@ -66,12 +66,12 @@ class ApiDefinitionFactory @Inject() (appConfig: AppConfig) extends Logging {
             version = Version3,
             status = buildAPIStatus(Version3),
             endpointsEnabled = appConfig.endpointsEnabled(Version3)
-          ),
-          APIVersion(
-            version = Version4,
-            status = buildAPIStatus(Version4),
-            endpointsEnabled = appConfig.endpointsEnabled(Version4)
           )
+//          APIVersion(
+//            version = Version4,
+//            status = buildAPIStatus(Version4),
+//            endpointsEnabled = appConfig.endpointsEnabled(Version4)
+//          )
```

Comment:
Perhaps leave an explanation. Something along the lines of:
```suggestion
// To be reinstated, see MTDSA-20412
//          APIVersion(
//            version = Version4,
//            status = buildAPIStatus(Version4),
//            endpointsEnabled = appConfig.endpointsEnabled(Version4)
//          )
```
See [MTDSA-20412](https://jira.tools.tax.service.gov.uk/browse/MTDSA-20412)